src/community/propose-a-component-or-pattern/index.md.njk

```diff
@@ -11,6 +11,7 @@ Anyone can propose a new component or pattern for the GOV.UK Design System.
 To be successful, proposals need to show that the component or pattern being suggested would be [useful and unique](/community/contribution-criteria).
 
 Follow 3 steps to propose a component or pattern for the Design System.
+Follow the 3 steps to propose a component or pattern for the Design System.
```

Comment:
Hey @OliverHannan!

It looks like you've got the correct change but it's added in addition to the previous line.

Could you edit this and replace the line 13?

You can use the following link to edit this:
https://github.com/OliverHannan/govuk-design-system/edit/patch-2/src/community/propose-a-component-or-pattern/index.md.njk?pr=%2Falphagov%2Fgovuk-design-system%2Fpull%2F771

Let us know if you need any help 👍

Reply:
Cheers, done now